### ISSUE TITLE
Phantom window in principal monitor

### ISSUE THREAD

**Issue by codePassion-dot** [fuhsjr00/bug.n#276]
I have two monitors, and the program it's working good, but when there is no windows in any of the two monitors the bar of the principal monitor say [2] or 1x1 |1. That means it is detecting some windows and there is not any window. In the second monitor is working good, when I don't have any window it says 1x1|-1 or [0] that's the expect behaivor. Some idea?

**Comment by joten** [comment]
You may try to identify the window, which bug.n falsely lists, by searching the window list (<kbd>Win</kbd><kbd>Shift</kbd><kbd>I</kbd>), or you may reset bug.n in this regard by closing it, deleting the C:\Users\<username>\AppData\Roaming\bug.n\data_WindowState.ini and restarting bug.n

**Comment by codePassion-dot** [comment]
Hello, thank you for your answer. I used the hotkey you told me and I found it was a  HwndWrapper window. I saw in another issues some rules to handle those windows, so I put this two rules to delete this windows in case their were in the windows list.
Rules:
Config_rule=HwndWrapper.LogiOverlay.*;;;1;0;0;0;0;0;close
Config_rule=;MainWindow.*;;1;0;0;0;0;0;close

The key in the rules is the action <close>, because if there is a hwnwrapper window in the list this action will close it. So it's not a distraction or a annoying thing anymore, thank you very much.

pd: Your project is amazing, I use it in my daily.
